templates/issues.html

```diff
@@ -26,7 +26,7 @@
                         </div>
                 </div>
             </div>
-            <div>Issues last refreshed from GitHub on {{last_updated}}.</div>
+            <div><i class="fa fa-clock-o"></i>&nbsp;Updated <span am-time-ago="last_updated"></span></div>
```

Comment:
My Spidey senses from the "old Web" tell me that hard-coded "&nbsp" -- especially in this day and age -- are bad form, but I can't, as of yet, back this up.  Is there a sexier, CSS-equivalent styling/attribute you can go with, and apply, here?